releases/v1.1.0.toml

```diff
@@ -0,0 +1,150 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes [Container Runtime Interface](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md).
+CRI is a new plugin which allows connecting the containerd daemon directly to a
+Kubernetes kubelet to be used as the container runtime. The CRI GRPC interface
+listens on the same socket as the containerd GRPC interface and runs in the same
+process.
+
+In addition to all of the stability and bug fixes backported to 1.0,
+1.1 includes...
+
+- CRI plugin
+- ZFS and AUFS snapshotter
+- Improvements to the `ctr` tool
+- Better support for multiple platforms
+- Cross namespace content sharing
+- Better mount cleanup
+- Support for disabling plugins
+- TCP debug address for remote debugging
+- Update to Go 1.10
+- Improvements to the garbage collector
+
+## CRI Plugin
+
+This release of `cri` is a native plugin of `containerd`. It is built into
```

Comment:
For clarity, can we mention that `cri` was formerly `cri-containerd`?

Reply:
The next paragraph mentions the former `cri-containerd` daemon. Not sure the project rename needs to be mentioned here, since it was just moved from `cri-containerd` -> `containerd/cri`. If we feel there is some confusion we can always update the Github part of the release notes later.